converter/shieldhit/parser.py

```diff
@@ -165,47 +119,91 @@ def save_configs(self, target_dir: str):
             beam_f.write(str(self.beam_config))
 
         with open(path.join(target_dir, 'mat.dat'), 'x') as mat_f:
-            mat_f.write(str(self.mat_config))
+            mat_f.write(self.geo_mat_config.get_mat_string())
 
         with open(path.join(target_dir, 'detect.dat'), 'x') as detect_f:
             detect_f.write(str(self.detect_config))
 
         with open(path.join(target_dir, 'geo.dat'), 'x') as geo_f:
-            geo_f.write(str(self.geo_config))
+            geo_f.write(self.geo_mat_config.get_geo_string())
+
+    def get_configs_json(self) -> dict:
+        """
+        Return a dict representation of the config files. Each element has
+        the config files name as key and its content as value.
+        """
+        configs_json = {
+            "beam": str(self.beam_config),
```

Comment:
add file extensions as well, this should be proper file names

Reply:
Should dict keys have '.' in them? That seems unusual to me. Can we swap the '.' for '_' (so it would be mat_dat, geo_dat etc.)?

Reply:
What can make a problem with `.` ? JS ? From python side its just a normal string